sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -0,0 +1,331 @@
+
+Admininistrator User Manual
+============================
+**Who is an "administrator"?**
+
+A person who :
+    * helps to plan and oversee a project using KA Lite
+    * installs and updates KA Lite
+    * can create coach logins, Learner logins, download videos and language packs
+
+Administrator Glossary
+-------------------------------------------
+For users that choose to register online, there are some important terms to familiarize yourself with so that you understand how the flow of data works between installations and the online data hub.
+
+**Sharing Network**
+	A sharing network is a group of devices that share data between one another. Data can mean video content, access software applications, and certain files.
+
+**Organization**
+	An organization is a group of people responsible for administering a set of Sharing Networks. An organization can have multiple administrators and manage multiple sharing networks.
+
+**Facility**
+	A facility is the physical space in which a device is located (e.g. a school or a computer lab in a community center). Learner, coach, and admin accounts are associated with a particular facility.
+
+**Device**
+	A device should be able to run a KA Lite server (most computers) and other devices to be used as clients. One common configuration is using a Raspberry Pi or other inexpensive computer as a server and relatively cheap tablets as client devices. Tablets can access the servers through an access point, such as a Wi-Fi dongle, or some other networking device such as a router.
+
+**Web Browser**
+    A program that retrieves and presents information resources on the World Wide Web. Popular web browsers include Internet Explorer, Google Chrome, Mozilla Firefox, and Safari.
+
+**Torrent**
+    A file or files sent using the BitTorrent protocol. It can be any type of file, such as a movie, song, game, or application. During the transmission, the file is incomplete and therefore referred to as a torrent. Torrent downloads that are incomplete cannot be opened as regular files, because they do not have all the necessary data.
+
+**Seeder**
+    Seeders are users who have a complete version of the file you wish to download. If there are no seeders for a particular file, you will not be able to download the file. Seeders are extremely important, for they help distribute the file.
+
+**Bandwidth**
+    The amount of data that an Internet connection can handle in a given time. An Internet connection with larger bandwidth can move a certain amount of data much faster than an Internet connection with a lower bandwidth.
+
+Running the KA Lite Server
+---------------------------
+After a successful installation of KA Lite, the terminal should give you a command to copy. It will look something like the following:
+
+    ``/PATH/TO/KALITE/bin/kalite start``
```

Comment:
Looks like we do use bin/kalite. Would it make sense then to drop the other calls to the setup scripts?